gtfs/gtfs_utils/gtfs_utils/core_computations.py

```diff
@@ -104,7 +107,8 @@ def compute_trip_stats(feed: ptg.feed,
     - ``trip_id`` - Trip identifier, as specified in `trips.txt` file.
     """
 
-    logging.info(f'Starting compute_trip_stats for {date} from {gtfs_file_name}')
+    source_files_str = ';'.join(source_files_base_name)
+    logging.info(f'Starting compute_trip_stats for {date} from {source_files_str}')
```

Comment:
In this case I would prefer printing the source file names/paths (one in each line), but I won't force you to.